### ISSUE TITLE
@Rotemblumberg:

### ISSUE THREAD

**Issue by RotemBlumberg** [gepruts/QuickSwitch#7]
@Rotemblumberg:

is this still an issue?

_Originally posted by @gepruts in https://github.com/gepruts/QuickSwitch/issues/3#issuecomment-1105593541_

It is. I appreciate your help.

**Comment by gepruts** [comment]
Please answer the following questions:


- Is QuickSwitch in a writable folder? (where new files can be created easily)
- Is there a JumpToFolder.ini in this folder?
- Can you post the contents of your JumpToFolder.ini?

**Comment by RotemBlumberg** [comment]
The app is located at the path to "C:\Users<Username>\pathTo\QuickSwitch\QuickSwitch.exe" with write permission availble.
The ini file is created (it is called: "QuickSwitch.ini" and not "JumpToFolder.ini").
This is its content after selecting "Allow AutoSwitch" in Word save dialog:
[Dialogs]
WINWORD.EXE___Save As=1
But the quick switch does not occur.
I click the other explorer folder and then back to the save dialog, but nothing happens.
I tried it on two different windows 11 machines.

Thanks.

**Comment by gepruts** [comment]
Sorry, I mixed up QuickSwitch and JumpToFolder. Not for the first time ...
<br>

QuickSwitch "calculates" the number of (hidden/normal) windows between the most recent used file manager and the file dialog. 
In 95% of the cases, this is 2 windows, like for example (1) Notepad's Open dialog, (2) Notepad and (3) File Explorer.

When you have AutoSwitch enabled, for the remaining 5%, you can follow these steps:

- Open the unwilling file dialog.
(Nothing will happen as QuickSwitch doesn't understand/ miscalculates)
- Press 'CTRL + Q'
- Select **AutoSwitch exception** from the QuickSwitch menu
- Follow the on-screen steps.
In short, this lets QuickSwitch figure out and learn what the correct "window-distance" is for this specific application.


Does following these steps solve your issue?
